content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -13,7 +13,7 @@ You can use the Kubernetes command line tool kubectl to interact with the API Se
 {{% capture body %}}
 ## docker run
 
-To run an nginx Deployment and expose the Deployment, see [kubectl run](/docs/reference/generated/kubectl/kubectl-commands/#run).
+To run an nginx Pod and expose the Pod, see [kubectl run](/docs/reference/generated/kubectl/kubectl-commands/#run).
```

Comment:
You can `kubectl create deployment` (eg as explained in https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive/)

I think it's at least worth explaining why this is using `kubectl run` (for example: to focus on a particular detail).

Reply:
I think `kubectl run` the same as `docker run`,it is similar parameters,but `kubectl create deployment` didn't.Add notes to remind users to create services available for the entire Cluster.

Reply:
I'd suggest we advocate Deployment rather than Pod in the docs, even when the user only wants to run a single Pod.

Given that the `kubectl create deployment` command is gonna [support replicas](https://github.com/kubernetes/kubernetes/commit/b6213ed931930eba27b4a53db54b4b4d224a1c59) in 1.19. I'd suggest we replace `kubectl run` with the equivalent `kubectl create deployment` command and target this PR to dev-1.19.

Reply:
I agree with https://github.com/kubernetes/website/pull/21536#discussion_r436363300

However I wonder about v1.18 in the historic docs, which will be around for a while. `kubectl run` works OK for 1.17, `kubectl create deployment` works for v1.19, but the v1.18 page is not correct.

_I don't know_ yet how to deal with that. How about: fix for v1.19 first, and then base the v1.18 fix on how the page looks for v1.19?

Reply:
so marge this to v1.18 ? and v1.19 use `kubectl create deployment`?
we have added [tips](https://github.com/kubernetes/website/pull/21536/files#diff-16decda489c4f33805ead292d4af10ceR46) to advocate the user that it is recommended to use deploymnet .In v1.18 , [`kubectl create deployment`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#-em-deployment-em-) not support replicas 、port and env flag.

Reply:
I'd like to see the shape of the page for v1.19 and then adapt that content to work with the limitations of v1.18

Reply:
thx for lot.Please review again.